dss/operations/checkout.py

```diff
@@ -151,37 +150,43 @@ def process_keys(self):
                 checkout_status[_key] = bundle_internal_status
             elif _key.startswith(FILE_PREFIX):
                 temp = collections.defaultdict(blob_checkout=False, should_be_cached=False)
-                file_metadata = self._verify_get(self.handle, self.replica.bucket, _key)
+                file_metadata = self._get_metadata(self.handle, self.replica.bucket, _key)
+                if not file_metadata:
+                    checkout_status[_key] = 'KEY_DOES_NOT_EXIST'
```

Comment:
nit: I would want to set the key to  `None` to avoid parsing issues if the output gets piped into another script.

Reply:
I could see that, though either could easily be parsed.  I chose this since it's unambiguous.

Reply:
`[k for k,v in checkout_status.items() if v is not None]` 
vs
`[k for k,v in checkout_status.items() if v !='KEY_DOES_NOT_EXIST ]`  

for the latter you would need to know that that's the value of they if  the file_metadata  is  not found. I don't think its an intuitive approach, it is explicit tho.  
🤷‍♂

Reply:
I can see your point.  I'd expect `None` when it's not in the checkout bucket, but this is when the file isn't even in the main bucket and so can't be checked out even if you wanted to.  I wanted to differentiate between those two cases, but I think I'll use `sys.stderr.write()` here too, as Brian suggested.